pkg/security/probe/field_handlers.go

```diff
@@ -156,6 +156,18 @@ func (fh *FieldHandlers) ResolveContainerID(ev *model.Event, e *model.ContainerC
 	return e.ID
 }
 
+// ResolveContainerCreatedAt resolves the container creation time of the event
+func (fh *FieldHandlers) ResolveContainerCreatedAt(ev *model.Event, e *model.Process) int {
+	if e.ContainerCreatedAt == 0 {
+		if entry, _ := fh.ResolveProcessCacheEntry(ev); entry != nil {
+			if cgroup, _ := fh.resolvers.CgroupsResolver.Get(entry.ContainerID); entry != nil {
```

Comment:
should we check that the entry has a ContainerID before trying to request the CgroupResolver ?

Reply:
wondering if the `e` process context is not already the process cache entry ? if so you could remove the `ResolveProcessCacheEntry` call